.github/workflows/contract-tests.yml

```diff
@@ -17,9 +17,24 @@ on:
       - reopened
       - ready_for_review
       - synchronize
+    paths:
+      - ".github/workflows/contract-tests.yml"
+      # Ensure these are synchronized with the paths in the check-changed-files job
+      - "*/src/**"
+      - "**/*.java"
+      - "**/*.gradle"
+      - "**/*.properties"
+      - "gradle*"
```

Comment:
Should we also check any openapi specs that might change as well.

Reply:
mmm, where are these stored? If within `*/src/**`, these are already covered. I can't see any other json files in the repo that would need to be tested via `./gradlew pactConsumerTests`. Unless I'm missing something?